src/dotnet-format.csproj

```diff
@@ -3,19 +3,23 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
-    <LangVersion>7.3</LangVersion> 
+    <LangVersion>7.3</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <RootNamespace>Microsoft.CodeAnalysis.Tools</RootNamespace>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
+
     <Description>Command line tool for formatting C# and Visual Basic code files based on .editorconfig settings.</Description>
-    <PackageIconUrl>https://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
```

Comment:
How is the icon generated? Does aracde do this automatically?

Reply:
No change with icon, just moved the line.